src/test/java/uk/ac/ebi/embl/api/validation/fixer/sourcefeature/CountryQualifierFixTest.java

```diff
@@ -72,6 +72,15 @@ public void testCheckCountryQualifierWithValidValue()
         assertEquals(0, check.check(feature).getMessages().size());
     }
 
+    @Test
```

Comment:
I wonder if we could have a unit test somehow to make sure we add all covid-19 qualifiers correctly (incl. note). Otherwise there is a good chance things might not work or will just stop working silently.